staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/service.proto

```diff
@@ -0,0 +1,54 @@
+// To regenerate service.pb.go run hack/update-generated-kms.sh
+syntax = "proto3";
+
+package v1beta1;
+
+// This service defines the public APIs for remote KMS provider.
+service KMSService {
```

Comment:
KMSService is redundant. I'd like to fix this this before 1.10. Service names are painful to change.